### PR TITLE
Polishing search bar

### PR DESCRIPTION
**Pull request by Duncid**
- Polishing the look and feel of the bar
- Adding an "erase" functionality

Note: I couldn't test the component "IRL".

bumping sparkle to 1.96

### CODE REVIEW

File: sparkle/src/components/Searchbar.tsx [hunk 1]
```diff
@@ -24,20 +27,30 @@ export function Searchbar({
   disabled?: boolean;
   className?: string;
 }) {
+  const inputRef = useRef<HTMLInputElement>(null);
```
Comment (philipperolet) [inline sparkle/src/components/Searchbar.tsx:30]: this inputref is not assigned so not useful; can be safely removed along with the two lines below referencing it

**Review by philipperolet — CHANGES_REQUESTED** [review]
Thanks for making it look better ! Code wise 2 little comments, otherwise LGTM :+1: 
Can help with the second one if you need

**Comment by philipperolet** [comment]
Two late additions:
1. current sparkle version is at 95, this PR can bump it to 96
2. did you want the focus to go to the searchbar on click on Xmark icon? can show you how to do this or do it if needed

**Comment by Duncid** [comment]
@philipperolet I bumped to 96.

> did you want the focus to go to the searchbar on click on Xmark icon? can show you how to do this or do it if needed

No I want the focus to leave the searchbar (which seem to be the case)

**Pull request merged**